controllers/metal3.io/baremetalhost_controller_test.go

```diff
@@ -1766,6 +1769,54 @@ func TestUpdateEventHandler(t *testing.T) {
 
 			expectedProcess: true,
 		},
+		{
+			name: "do-requeue-when-finalizers-are-different",
```

Comment:
Isn't already covered by https://github.com/metal3-io/baremetal-operator/pull/1881/files#diff-20c1a976622b8d54cf98349e3e6219778fa00e89ed669f1bb12cd778326469bbR1733?

Reply:
I can remove this but this was to simulate that this would requeue despite the LastUpdated in the status.  This wouldn't do anything other than prove that this still requeues and not get swallowed by the false requeue from the LastUpdated in the status.

Reply:
Resolving is the first test is removed from patch set.